service/controller/appcatalog/v1/resource/index/create.go

```diff
@@ -0,0 +1,55 @@
+package index
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+	cm, err := toConfigMap(createChange)
```

Comment:
```suggestion
	configMap, err := toConfigMap(createChange)
```

I'd prefer it if we used configMap rather than cm. This then aligns with using currentConfigMap etc in newCreateChange.